src/miner.cpp

```diff
@@ -600,7 +600,6 @@ bool CreateCoinStake( CBlock &blocknew, CKey &key,
 
             txnew.vout.push_back(CTxOut(0, CScript())); // First Must be empty
             txnew.vout.push_back(CTxOut(nCredit, scriptPubKeyOut));
-            //txnew.vout.push_back(CTxOut(0, scriptPubKeyOut));
```

Comment:
:+1:

Reply:
You left that code in there commented out, probably from the original write of V8. I figured it should go away.